airbyte-cdk/bulk/toolkits/load-gcs/build.gradle

```diff
@@ -3,5 +3,7 @@ dependencies {
     implementation project(':airbyte-cdk:bulk:core:bulk-cdk-core-load')
     api project(':airbyte-cdk:bulk:toolkits:bulk-cdk-toolkit-load-object-storage')
 
+    implementation("com.google.cloud:google-cloud-storage:2.50.0")
```

Comment:
you probably want to rebase against latest `gcs_load_toolkit` - I actually added a `load-gcp` module, which declares the GCP BOM